src/index.ts

```diff
@@ -327,6 +334,21 @@ export const createInstance = (config: Config): Instance => {
       uploadObjectAsJson: async (
         object: Record<string, unknown>
       ): Promise<string> => apiClient.uploadObject(object),
+
+      startUrlUploadJob: (urls: string[]): Promise<UrlToJobId[]> | InvalidInputMessage => {
+        if(urls.length>100){
```

Comment:
This should throw an error instead of returning a different type. If you return two types you require the user to interrogate the return type to see if there is a `errorMessage` field on it, and if there is, then they know it's `InvalidInputMessage`.

If you throw an error, all they have to do is `try/catch` the call and if it catches, then it was an error and they can parse the message.

If some sort of parameter is wrong, or a failure happens inside of a function, throw an error instead of returning a special value. (Unless we've established a pattern otherwise)

Reply:
done!